static/js/subs.js

```diff
@@ -113,6 +113,15 @@ exports.toggle_pin_to_top_stream = function (sub) {
     stream_edit.set_stream_property(sub, 'pin_to_top', !sub.pin_to_top);
 };
 
+exports.update_is_stream_admin = function (sub, value) {
+    sub.is_stream_admin = value;
+    stream_data.update_calculated_fields(sub);
+    var node = $(".stream-row[data-stream-id='" + sub.stream_id + "']");
+    if (node.hasClass("active")) {
+        stream_edit.show_settings_for(node);
+    }
```

Comment:
Rerender the whole page of stream, cause if user is removed/add to stream admin, stream page should be changed. This will rerender page only if stream is active(stream page is open).